src/main/java/org/junit/rules/RuleChain.java

```diff
@@ -77,8 +77,12 @@ private RuleChain(List<TestRule> rules) {
      *
      * @param enclosedRule the rule to enclose.
      * @return a new {@code RuleChain}.
+     * @throws NullPointerException if the argument {@code enclosedRule} is null
```

Comment:
Please change this line to `@throws NullPointerException if the argument {@code enclosedRule} is {@code null}